app/components/Image/ProfilePicture.tsx

```diff
@@ -1,16 +1,18 @@
 import type { UserEntity } from 'app/reducers/users';
 import CircularPicture from './CircularPicture';
+import type { ComponentProps } from 'react';
```

Comment:
Is this not sorted wrongly? Why is it not caught by our linter?

Reply:
heh, our sorting rules are a bit _off_. Everything is sorted automatically, so this is how it wants it. 
I think the `app/` group overrides type imports :/ Should look at it, but not in this PR

Reply:
Yes, definitely out of scope here.